strassengezwitscher/strassengezwitscher/settings/default.py

```diff
@@ -117,6 +118,15 @@
     'auth.user': lambda u: '/intern/users/%s/' % u.id,
 }
 
+######################################################################################################
+#                                           Recaptcha settings
+######################################################################################################
+
+# This is key for development purposes only
+RECAPTCHA_SECRET_KEY = ['6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe']
```

Comment:
Wenn das nur fürs die Entwicklung ist, wäre es dann nicht besser in `settings/dev.py` aufgehoben?